docs/get-started.md

```diff
@@ -78,7 +84,8 @@ Jon Snow
 
 By now you should know that we need to **define a Match**.
 
-With your favourite text editor, open the `.espanso/default.yml` file, introduced previously in the [Configuration](#configuration) section. You should see something like:
+With your favourite text editor, open the `.espanso/default.yml` file, introduced previously in the 
```

Comment:
Could you please change `.espanso/default.yml` to `espanso/default.yml`? The dot is a left over from the first versions